File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.cs
using System;

public class Solution {
    public int solution(int[] num_list) 
    {
        string even = "";
        string odd = "";
        
        foreach(int n in num_list)
        {
            if(n % 2 == 0)
                even += n.ToString();
            else
                odd += n.ToString();
        }
        
        return int.Parse(even) + int.Parse(odd);
    }
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    
    int even = 0;
    int odd = 0;
    for (int i : num_list)
    {
        if (i % 2 == 0)
        {
            even *= 10;
            even += i;
        }
        else
        {
            odd *= 10;
            odd += i;
        }
    }
    
    return even + odd;
}